src/pages/review.astro

```diff
@@ -58,7 +58,7 @@ const isVisible = true;
 >
   <div slot="headerChild" class="mt-8 lg:mt-16">
     <h1
-      class="text-center font-header text-4xl text-white md:text-6xl lg:text-8xl"
+      class="text-center font-header text-4xl font-bold text-white md:text-6xl lg:text-8xl"
```

Comment:
If font-bold is added to all headers you could have just added a common style to all headers in the global CSS file

Reply:
yes,can do that. but the issue is not just with header. previously we just had one font size for header, now we need to switch between bold and medium. that's why i had to add so many font-bolds at once